.circleci/config.yml

```diff
@@ -28,14 +31,29 @@ commands:
     steps:
       - run:
           name: 'Solidus <<parameters.branch>>: Generate Gemfile.lock'
-          command: bundle lock
+          command: |
+            rm Gemfile
+
+            cat \
```

Comment:
What about adding those things in the Gemfile directly. That Gemfile was used to run bundle locally and consider the dependencies specified in the gemspec. But now that we are going with using this project as a generator only I think we don't need that content anymore.